diff -urN genius-0.7.0.orig/configure.in genius-0.7.0/configure.in
--- genius-0.7.0.orig/configure.in	2004-08-12 18:27:34.697762936 +0200
+++ genius-0.7.0/configure.in	2004-08-12 18:27:51.248246880 +0200
@@ -43,7 +43,7 @@
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE")
 AC_SUBST(GETTEXT_PACKAGE)
 
-ALL_LINGUAS="az ca cs de el en_CA en_GB es fr ga hi hr it ko nl no pt pt_BR sq sr sr@Latn sv pl zh_CN"
+ALL_LINGUAS="az ca cs de el en_CA en_GB es fr ga hi hr it ko nb nl pt pt_BR sq sr sr@Latn sv pl zh_CN"
 AM_GLIB_GNU_GETTEXT
 
 dnl ===============================================================================
