diff -urN aa/genius-0.7.2/src/genius.c genius-0.7.2/src/genius.c
--- aa/genius-0.7.2/src/genius.c	2004-12-22 02:44:15.000000000 +0100
+++ genius-0.7.2/src/genius.c	2005-02-04 19:22:08.764857664 +0100
@@ -35,18 +35,6 @@
 #include <stdio.h>
 #include <locale.h>
 
-#ifdef USE_NCURSES
-#ifdef INC_NCURSES
-#include <ncurses/curses.h>
-#include <ncurses/term.h>
-#else
-#include <curses.h>
-#include <term.h>
-#endif
-#else
-#include <termcap.h>
-#endif
-
 #include "calc.h"
 #include "eval.h"
 #include "util.h"
@@ -62,6 +50,18 @@
 #include <readline/readline.h>
 #include <readline/history.h>
 
+#ifdef USE_NCURSES
+#ifdef INC_NCURSES
+#include <ncurses/curses.h>
+#include <ncurses/term.h>
+#else
+#include <curses.h>
+#include <term.h>
+#endif
+#else
+#include <termcap.h>
+#endif
+
 /*Globals:*/
 
 /*calculator state*/
