--- genius-1.0.10/gtkextra/Makefile.am~	2009-04-20 01:44:42.000000000 +0200
+++ genius-1.0.10/gtkextra/Makefile.am	2010-09-09 09:03:01.561368767 +0200
@@ -68,7 +68,7 @@
 libgtkextra_genius_a_SOURCES =	\
 	$(public_h_sources)	\
 	gtkextra-marshal.c	\
-	gtkextra-marshal.h	\
+	gnntkextra-marshal.h	\
 	gtkextratypebuiltins.c	\
 	gtkextratypebuiltins.h	\
 	gtkextra.c		\
@@ -99,11 +99,11 @@
 	gtkplotsurface.c
 
 $(srcdir)/gtkextra-marshal.h: gtkextra-marshal.list
-        ( @GLIB_GENMARSHAL@ --prefix=gtkextra gtkextra-marshal.list --header > gtkextra-marshal.tmp \
+	( @GLIB_GENMARSHAL@ --prefix=gtkextra gtkextra-marshal.list --header > gtkextra-marshal.tmp \
         && mv gtkextra-marshal.tmp gtkextra-marshal.h ) \
         || ( rm -f gtkextra-marshal.tmp && exit 1 )
 $(srcdir)/gtkextra-marshal.c: gtkextra-marshal.h
-        ( @GLIB_GENMARSHAL@ --prefix=gtkextra gtkextra-marshal.list --body > gtkextra-marshal.tmp \
+	( @GLIB_GENMARSHAL@ --prefix=gtkextra gtkextra-marshal.list --body > gtkextra-marshal.tmp \
         && mv gtkextra-marshal.tmp gtkextra-marshal.c ) \
         || ( rm -f gtkextra-marshal.tmp && exit 1 )
 
