--- genius-0.7.6.1/configure.in~	2007-01-04 01:44:24.000000000 +0100
+++ genius-0.7.6.1/configure.in	2007-01-04 17:48:41.000000000 +0100
@@ -58,8 +58,8 @@
 
 dnl Check for termcap
 AC_CHECK_LIB(termcap, tgetent,[TERMCAP_LIB="-ltermcap"],
-	     [AC_CHECK_LIB(ncurses, tgetent,
-                           [TERMCAP_LIB="-lncurses"
+	     [AC_CHECK_LIB(tinfo, tgetent,
+                           [TERMCAP_LIB="-ltinfo"
                             AC_DEFINE(USE_NCURSES,[1], [Use ncurses library])
                             AC_CHECK_HEADER(ncurses/curses.h, [
 					    AC_DEFINE(INC_NCURSES,[1],[ncurses include])])], [
