diff -urN genius-0.7.1.orig/src/gnome-genius.desktop.in genius-0.7.1/src/gnome-genius.desktop.in
--- genius-0.7.1.orig/src/gnome-genius.desktop.in	2004-11-30 21:24:23.058590960 +0100
+++ genius-0.7.1/src/gnome-genius.desktop.in	2004-11-30 21:25:39.532965088 +0100
@@ -4,7 +4,7 @@
 Exec=gnome-genius
 Icon=gnome-calc2.png
 Terminal=false
-Categories=Application;GNOME;Scientific;Math;Office;
+Categories=GTK;GNOME;Education;Science;Math;
 Mime-Type=text/x-genius;
 _Name=Genius Math Tool
 _Comment=Genius Mathematical Tool and Calculator
