diff -urN genius-1.0.2/src/gnome-genius.desktop.in genius-1.0.2.new/src/gnome-genius.desktop.in
--- genius-1.0.2/src/gnome-genius.desktop.in	2007-11-20 09:35:37.000000000 +0100
+++ genius-1.0.2.new/src/gnome-genius.desktop.in	2007-11-22 16:01:30.000000000 +0100
@@ -4,7 +4,8 @@
 Exec=gnome-genius
 Icon=gnome-genius
 Terminal=false
-Categories=GNOME;Science;Math;Education;Calculator;Utility;
+Categories=GTK;GNOME;Education;Science;Math;
 MimeType=text/x-genius;
 _Name=Genius Math Tool
 _Comment=Genius Mathematical Tool and Calculator
+# vi: encoding=utf-8
