diff -urN aa/genius-0.7.0/gtkextra/gtkplot3d.h genius-0.7.0/gtkextra/gtkplot3d.h
--- aa/genius-0.7.0/gtkextra/gtkplot3d.h	2004-08-04 17:29:47.000000000 +0200
+++ genius-0.7.0/gtkextra/gtkplot3d.h	2004-08-13 00:40:28.190260936 +0200
@@ -235,10 +235,10 @@
                                                  gint offset);
 gint 		gtk_plot3d_get_titles_offset 	(GtkPlot3D *plot);
 void            gtk_plot3d_axis_set_scale       (GtkPlot3D *plot,
-                                                 GtkPlotAxisPos axis,
+                                                 GtkPlotOrientation axis,
                                                  GtkPlotScale scale);
 GtkPlotScale    gtk_plot3d_axis_get_scale       (GtkPlot3D *plot,
-                                                 GtkPlotAxisPos axis);
+                                                 GtkPlotOrientation axis);
 /* Grids */
 void            gtk_plot3d_major_grids_set_visible    (GtkPlot3D *plot,
 						       gboolean x,
